pkg/apis/admission/types.go

```diff
@@ -92,6 +92,11 @@ type AdmissionResponse struct {
 	// PatchType indicates the form the Patch will take. Currently we only support "JSONPatch".
 	// +optional
 	PatchType *PatchType
+	// Annotations is an unstructured key value map set by remote admission controller (e.g. error=image-blacklisted).
```

Comment:
Add more about the intent, e.g. `Annotations may be provided by the admission webhook to add additional context to the audit log for this request`

Reply:
Done.